cypress/integration/chi-custom-elements/brand.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('Brand', () => {
+  before(() => {
+    cy.visit('tests/custom-elements/brand.html');
+  });
+
+  beforeEach(() => {
+    ['brand-base-default', 'brand-inverse-sm', 'brand-white-md', 'brand-black-lg', 'brand-black-default', 'brand-white-xxl'].forEach((s) => {
+      cy.get(`[data-cy=${s}]`, { timeout: 5000 })
+        .should('have.class', 'hydrated')
+        .children()
+        .first()
+        .should('match', 'a.a-brand')
+        .as(s);
+    });
+  });
+
+  it('Base brand should be included inside the custom element', () => {
+    cy.get('@brand-base-default')
```

Comment:
I'm not sure that is what I suggested but is indeed a good starting point. Now you can define a function just like this: 
`const idChecker = function(alias, id) {
    cy.get(alias)
        .children()
        .first()
        .should('have.id', id);
}
`
and call it whenever you need with the new defined aliases. For example: idChecker('@brand-base-default', 'brand-base');

Reply:
After we get hold of the element it might not always be the same scenarios for assertion if you see last two tests don't need the child el and maybe in other I might do some other logic, So I kind of feel it's unnecessary, I don't know but I do want it to be consistent across all of tests. So what are we agreeing to @dani-cl-madrid @SergioQCL.

I like the assertions to be independent.

Reply:
Maybe the example was not clear enough, but the function was only intended for covering assertions doing exactly the same steps, not for all of them. In your case it could be applied 4 times, that's why I considered it could be useful :)